test/unit/kernel/src/KernelLambda.cpp

```diff
@@ -30,6 +30,10 @@ void operator()()
     alpaka::test::KernelExecutionFixture<TAcc> fixture(
         alpaka::vec::Vec<Dim, Idx>::ones());
 
+#if BOOST_COMP_MSVC || defined(BOOST_COMP_MSVC_EMULATED)
+    #pragma warning(push)
+    #pragma warning(disable: 4702)  // warning C4702: unreachable code
```

Comment:
I am not yet sure why visual studio 2019 thinks that there is unused code in here...